draft-ietf-capport-api.md

```diff
@@ -93,7 +93,7 @@ The following keys are defined at the top-level of the JSON structure returned b
 - "captive" (required, boolean): indicates whether the client is in a state of captivity, i.e it has not satisfied the conditions to access the external network. If the client is captive (i.e. captive=true), it can still be allowed enough access for it to perform server authentication {{server-auth}}.
 - "user-portal-url" (optional, string): provides the URL of a web portal with which a user can interact.
 - "venue-info-url" (optional, string): provides the URL of a webpage or site on which the operator of the network has information that it wishes to share with the user (e.g., store info, maps, flight status, or entertainment).
-- "expire-date" (optional, string formatted as {{!RFC3339}} datetime): indicates the date and time after which the client will be in a captive state. The API server SHOULD include this value if the client is not captive (i.e. captive=false) and SHOULD omit this value for captive clients.
+- "seconds-remaining" (optional, integer): indicates the number of seconds remaining, after which the client will be placed into a captive state. The API server SHOULD include this value if the client is not captive (i.e. captive=false) and SHOULD omit this value for captive clients.
```

Comment:
given that now its "seconds remaining", should  we state that the API  server SHOULD set this value to 0 if the client is captive to be more explicit ?

Reply:
I would have thought that omitting this would be better.

Reply:
I think omitting is indeed more clear, as it already suggests.

Reply:
ok, that's fine too. My thought was having the value of zero is a more explicit indication given that its an optional field to begin with, however i don't have that strong feelings about it. If both of you think omitting is better, i'm good with it.